src/gc_lock.rs

```diff
@@ -126,7 +126,13 @@ mod test {
     fn completed_backup_ok() {
         let archive = ScratchArchive::new();
         let source = TreeFixture::new();
-        backup(&archive, &source.live_tree(), &BackupOptions::default()).unwrap();
+        backup(
+            &archive,
+            &source.live_tree(),
+            &BackupOptions::default(),
+            None,
```

Comment:
Aside: One thing I love about this pattern is that we can later write tests about what steps were taken.